app/src/main/res/values-it/about.xml

```diff
@@ -0,0 +1,615 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="about_title">Informazioni su...</string>
```

Comment:
@m2049r is this string too long?

Reply:
Another possibility I found on the internet is to use '?'

Reply:
maybe just `Info`?

Reply:
it turns out this string is not used anywhere any more - drop it.

Reply:
Sorry @m2049r, does "drop" mean "remove" or "leave it as it is"?

Reply:
Ok, string removed.